lib/modules/showImages/expando.js

```diff
@@ -54,13 +52,13 @@ export class Expando {
 	static getEntryExpandoFrom(thing: ?Thing): ?Expando {
 		if (!thing) return null;
 
-		const button = $(thing.entry).find('> .expando-button')[0];
+		const button = thing.entry.querySelector('.expando-button');
```

Comment:
This is safe because if there are multiple expando-buttons, the first one will be the parent (self post)